packages/auth/Dockerfile

```diff
@@ -6,7 +6,9 @@ ENV NPM_CONFIG_LOGLEVEL warn
 
 # Install npm dependencies first (so they may be cached if dependencies don't change)
 COPY package.json package.json
+COPY packages/auth/tsconfig.json packages/auth/tsconfig.json
```

Comment:
Why were this added? 

1. I wanted to make adding Elastic APM monitoring for all apps as easy as possible, so `import '@opencrvs/commons/monitoring'`
2. If we look at the build `commons` package, the actual location of `monitoring.js` would actually be `@opencrvs/commons/build/dist/monitoring.js`. To make the path tidier, I had to configure `"exports"` in common's package.json (Introduced in node 12.x, so not exactly cutting edge). 
3. When I was building the package, I noticed `@ef-carbon/tspm` is so outdated, that it didn't support the "exports" field, so I chose to get rid of it completely, and use `ts-config-paths/register` also when running the apps in production. `ts-config-paths` requires tsconfig.json to be around, so there's your answer!